internals/daemon/daemon.go

```diff
@@ -795,6 +796,13 @@ func New(opts *Options) (*Daemon, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Propagate a valid plan (empty is also valid) to the subscribed
+	// overlord managers.
+	planMgr := ovld.PlanManager()
+	err = planMgr.Load()
+	if err != nil {
+		return nil, err
```

Comment:
No need to wrap the error here, as we want to report the error from the plan manager directly to the caller, right?

Reply:
I typically wrap an error if it adds context that could be useful when interpreting the error, especially if its a library kind of call that could be use from multiple contexts. I will give this another thought, but I suspect the error context will be clear by the time it exists a public overlord manager method (or else I need to fix it).

Reply:
Thanks! I fixed this and improved the errors from the other managers initialised on overlord.New(). Resolving.

Reply:
Since the error comes from the internals/plan library, I wrapped it with something more plan related. The internal error could be "validation failed", but it would be useful to know the plan manager failed to load the plan.

Reply:
Resolving.